### PR TITLE
Use new SE API to check if the conversation access can be managed

### PR DESCRIPTION
**Pull request by mikeger**
- Cleaner implementation, tested and implemented in one place.

### CODE REVIEW

**Comment by daehn** [comment]
@mikeger there is a similar check in `ParticipantsCellViewModel`.

File: Wire-iOS/Sources/UserInterface/Conversation/Content/Cells/ParticipantsCell/ParticipantsCellViewModel.swift [hunk 1]
```diff
@@ -99,10 +99,9 @@ struct ParticipantsCellViewModel {
     }
     
     var showInviteButton: Bool {
-        guard case .started = message.actionType else { return false }
-        return true == message.conversation?.allowGuests
-            && nil != message.conversation?.team
-            && ZMUser.selfUser().isTeamMember
+        guard case .started = message.actionType,
+                let conversation = message.conversation else { return false }
+        return conversation.canManageAccess
```
Comment (mikeger) [inline Wire-iOS/Sources/UserInterface/Conversation/Content/Cells/ParticipantsCell/ParticipantsCellViewModel.swift]: @daehn please confirm that I am holding it right.

Comment (mikeger) [reply Wire-iOS/Sources/UserInterface/Conversation/Content/Cells/ParticipantsCell/ParticipantsCellViewModel.swift]: @daehn bump

Comment (daehn) [reply Wire-iOS/Sources/UserInterface/Conversation/Content/Cells/ParticipantsCell/ParticipantsCellViewModel.swift]: Yes, looks correct to me!

**Pull request merged**